gadget/mountedfilesystem.go

```diff
@@ -375,7 +379,27 @@ func (f *mountedFilesystemUpdater) Update() error {
 		}
 	}
 
-	if skipped == len(f.ps.ResolvedContent) {
+	deleted := false
+	if f.fromPs != nil {
+		for _, c := range f.fromPs.VolumeStructure.Content {
+			if knownContent[c.Target] {
+				continue
+			}
+			destPath, _ := f.entryDestPaths(f.mountPoint, "", c.Target, "")
+			if osutil.IsDirectory(destPath) {
+				continue
```

Comment:
this doesn't seem to be covered by unit tests

Reply:
Done